### PR TITLE
feat: icon sidebar

### PR DESCRIPTION
**Pull request by songwongtp**
## Describe your changes
- Add alternative sidebar with only icons

### CODE REVIEW

File: src/lib/layout/Navbar.tsx [hunk 1]
```diff
@@ -37,194 +49,323 @@ interface MenuInfo {
...
+        <Flex justifyContent="space-between" alignItems="center">
+          {item.category === "Overview" && (
+            <Tooltip
+              label="Expand"
+              hasArrow
+              placement="right"
+              bg="honeydew.darker"
+            >
+              <IconButton
+                aria-label="overview"
```
Comment (bkioshn) [inline src/lib/layout/Navbar.tsx:178]: Add `_hover={{ bg: pebble.700, borderRadius: "full" }}`

File: src/lib/layout/Navbar.tsx [hunk 2]
```diff
@@ -37,194 +49,323 @@ interface MenuInfo {
...
+              label={submenu.name}
+              hasArrow
+              placement="right"
+              bg="honeydew.darker"
+            >
+              <Flex
+                cursor="pointer"
+                p={1}
+                m={2}
+                _hover={{ bg: pebble800, borderRadius: "8px" }}
```
Comment (bkioshn) [inline src/lib/layout/Navbar.tsx:208]: all hover should be pebble.700

File: src/lib/layout/Navbar.tsx [hunk 3]
```diff
@@ -37,194 +49,323 @@ interface MenuInfo {
...
-                key={submenu.slug}
-                onClick={() => AmpTrack(AmpEvent.USE_SIDEBAR)}
+          </AppLink>
+        ))}
+      </Box>
+    ))}
+  </Box>
+);
+
+const IconNavMenu = ({
```
Comment (bkioshn) [inline src/lib/layout/Navbar.tsx:150]: Might consider moving `IconNavMenu` and `FullNavMenu` to its own file.

**Review by bkioshn — COMMENTED** [review]
There should be x padding for divider in icon navigation.

**Review by evilpeach — APPROVED** [review]

**Review by bkioshn — APPROVED** [review]

**Pull request merged**